hack/.golint_failures

```diff
@@ -796,6 +796,7 @@ test/e2e/scalability
 test/e2e/scheduling
 test/e2e/servicecatalog
 test/e2e/storage
+test/e2e/storage/vsphere
```

Comment:
Please fix the lint errors. Only generated files should be added to exception list at this point.

Reply:
From @rohitjogvmw offline:

```
if I try removing dot imports I need to have something like ..
gomega.Expect(err).NotTo(gomega.HaveOccurred())
which is not readable.
I guess that is why test/e2e/storage is added in go lint exception.
We will loose the readability of the test code if we want to fix golint failures
```

This is a valid argument. I withdraw my comment.

Reply:
Hi Saad, currently test code uses dot imports.

. "github.com/onsi/ginkgo"
. "github.com/onsi/gomega"

golint complains about these dot imports.
If I remove the dot import then we will loose the readability of the test code.

Sample code before:
Expect(err).NotTo(HaveOccurred())

Sample code after removing dot from import:
gomega.Expect(err).NotTo(gomega.HaveOccurred())

Is it possible not to loose test code readability and also fix golint?